evaluation/create_api_usecase.md

```diff
@@ -0,0 +1,89 @@
+## Create An API
+### Primary Actor: Advanced User
+
+### Trigger
+The user wants to use a specific, not included API to solve a given task using a given function.
+
+### Preconditions
+1. The user is logged in.
+2. There is at least one (built-in or previously created via the Developer Suite) function available for the user's account.
+
+### Postconditions
+1. The API is added to the database.
+
+### Main success scenarion
+1. The user clicks on "Switch to Developer Suite".
+2. The user confirms the warning pop-up to enter the advanced section Developer Suite.
+3. The user switches to the second tab "Manage APIs".
+4. The user switches to the second sub-tab "Create a new API".
+5. The user unlocks the protection mode to enter the edit mode.
+6. The user specifies general information about the API, information about the connection and additional parameter, required for the API to work. This step is highly dependent on several circumstances which are based on the selected API and its capabilities or requirements in terms of required request and response.
+7. The user saves and finishes the API creation process.
+
+### Evaluation
+#### Task
+You, as an advanced user, want to add your favourite finance API for receiving current stock prices, **Yahoo Finance**, to your account.
+Create a new API based on the following information:
+
+Map the new API to the function `stock_price_by_name`. The name is `My Evaluation Yahoo Finance API` and its priority is `High`. As we want to use the API, it should be active. The URL which belongs to the API is `https://apidojo-yahoo-finance-v1.p.rapidapi.com/stock/get-detail`. The mandatory header data will be provided for your evaluation task separately.
```

Comment:
Die Funktion in den Fixtures, die automatisch geladen werden heißt `stock_price` und kann hierfür benutzt werden, würde ich also überall ändern oder willst du eine zweite Funktion stock_price_by_name erstellen extra für die Task?